src/plus/focus/focus.ts

```diff
@@ -258,6 +263,11 @@ export class FocusCommand extends QuickCommand<State> {
 				newlyConnected = Boolean(connected);
 			}
 
+			const result = yield* ensureAccessStep(state, context, PlusFeatures.Focus);
+			if (result === StepResultBreak) continue;
+
+			context.showGraduationPromo = getApplicablePromo(result.subscription.current.state, 'launchpad') != null;
```

Comment:
Is it intentional that we are doing this access check/step after the "connect an integration" step? Once this becomes a Pro feature, should we be doing the access confirmation first? That way we don't let a user go all the way through with creating an account and connecting an integration, only to block them afterward?

Reply:
Yeah it's intentional, as for most they will connect an integration which starts the trial.